src/formatDate.js

```diff
@@ -8,7 +8,47 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const split = fromFormat[3];
```

Comment:
The variable 'split' is not necessary, as you can directly use 'fromFormat[3]' when calling 'split'.